website/blog/2023-06-11-reborn.md

```diff
@@ -0,0 +1,56 @@
+---
+title: ReasonReact is back (now with Melange)
+---
+
+Since the ReScript project diverged from ReasonML and moved ReasonReact into `@rescript/react`, ReasonReact has been in limbo. While this happened, the Melange team released the first major release of [Melange](https://buttondown.email/anmonteiro/archive/melange-hits-v10/) and made the first steps towards a deeper integration with the OCaml ecosystem.
+
+We are happy to announce that ReasonReact is back and will be maintained and developed by the Melange team.
+
+## What this means
+
+We released ReasonReact to the [opam registry](https://opam.ocaml.org/packages/reason-react) and be installed with opam or esy if you prefer (even though Melange team recommends using opam).
+
+We updated the documentation to reflect these new changes and we will continue to improve it over time.
+
+## Changes
```

Comment:
```suggestion
## Full changelog
```

Reply:
I propose moving this section last.

Once I looked at a "full changelog" I thought it was the last section and I wouldn't read past it.